robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot

```diff
@@ -0,0 +1,95 @@
+    ...                  BillingState=California
+    ...                  BillingPostalCode=${faker.postcode()}
+    Set suite variable   &{ACCOUNT}
+    ${date} =            Get Current Date         result_format=%Y-%m-%d
+    Set suite variable   ${DATE}
+    &{CONTACT} =         API Create Contact
+    ...                  FirstName=${faker.first_name()}
+    ...                  LastName=${faker.last_name()}
+    ...                  npe01__HomeEmail__c=test@example.com
+    Set suite variable   &{CONTACT}
+
+*** Test Cases ***
+Create Gift For Contact Using New Gift Button
+    [Documentation]                      Open a contact record and click on New Gift button from the actions menu of the page.
+    ...                                  Verify that Single Gift form is loaded with prepopulated contact info and create a gift.
+    ...                                  Verify that opp page is loaded on save. Verify that new opportunity and payment are created with right info
+    [tags]                               unstable      feature:GE                    W-039584
+    Go To Page                           Details              Contact       object_id=${CONTACT}[Id]
+    Current Page Should Be               Details              Contact
+    Click Special Object Button          New Gift
+    Current Page Should Be               Form                          Gift Entry
+    Verify Field Default Value
+    ...                                  Existing Donor Contact=${CONTACT}[Name]
+    ...                                  Contact First Name=${CONTACT}[FirstName]
+    ...                                  Contact Last Name=${CONTACT}[LastName]
+    ...                                  Contact Personal Email=${CONTACT}[npe01__HomeEmail__c]
+    Fill Gift Entry Form
+    ...                                  Donation Date=Today
+    ...                                  Donation Amount=150
+    Click Button                         Save
+    #verify new opportunity and its payment are created with correct info
+    Current Page Should Be               Details              Opportunity
+    ${newopp_id}                         Save Current Record ID For Deletion    Opportunity
+    Verify Expected Values               nonns                Opportunity       ${newopp_id}
+    ...                                  Amount=150.0
+    ...                                  CloseDate=${DATE}
+    ...                                  StageName=Closed Won
+    &{payment} =  API Query Record       npe01__OppPayment__c      npe01__Opportunity__c=${newopp_id}
+    Verify Expected Values               nonns                          npe01__OppPayment__c    ${payment}[Id]
+    ...                                  npe01__Payment_Amount__c=150.0
+    ...                                  npe01__Payment_Date__c=${DATE}
+    ...                                  npe01__Paid__c=True
+
+Create Gift For Account Using New Gift Button
```

Comment:
Did you want the two tests in the same robot file ?

Reply:
they are both testing the same feature but on different objects and independent so thought to keep them together